experimental/handoff-library/csharp_dotnetcore/samples/LPProxyBot/LiverPersonConnector/LivePersonConnector.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace LivePersonConnector
+{
+    public interface ICredentialsProvider
```

Comment:
There is an ICredentialProvider core interafce in the SDK and folks might get confused. Also, given that these are properties, do we need an interface at all? Will you or anybody provide different implementations of a property-only class without behavior?

Reply:
Renamed. This is used by the connector but implemented by the user for separation of concerns. A struct would also work but the user (bot) still has to instantiate and pass into the connector, so it's not really much simpler.